src/video.js

```diff
@@ -64,3 +69,59 @@ export const checkVideoBidSetup = hook('sync', function(bid, adUnit, videoMediaT
 
   return true;
 }, 'checkVideoBidSetup');
+
+export function registerVastTrackers(moduleType, moduleName, trackerFn) {
+  if (typeof trackerFn == 'function') { vastTrackers.push({'moduleType': moduleType, 'moduleName': moduleName, 'trackerFn': trackerFn}); }
+}
+
+export function insertVastTrackers(trackers, vastXml) {
+  const doc = new DOMParser().parseFromString(vastXml, 'text/xml');
+  const wrappers = doc.querySelectorAll('VAST Ad Wrapper, VAST Ad InLine');
+  try {
+    if (wrappers.length) {
+      wrappers.forEach(wrapper => {
+        if (trackers.get('impression')) {
+          trackers.get('impression').forEach(trackingUrl => {
+            const impression = doc.createElement('Impression');
+            impression.appendChild(doc.createCDATASection(trackingUrl));
+            wrapper.appendChild(impression)
+          });
+        }
+      });
+      vastXml = new XMLSerializer().serializeToString(doc);
+    }
+  } catch (error) {
+    logError('an error happened trying to insert trackers in vastXml');
+  }
+  return vastXml;
+}
+
+export function getVastTrackers(bid) {
+  let trackers = [];
+  vastTrackers.filter(
+    ({moduleType, moduleName, trackerFn}) => isActivityAllowed(ACTIVITY_REPORT_ANALYTICS, activityParams(moduleType, moduleName))
+  ).forEach(({trackerFn}) => {
+    let trackersToAdd = trackerFn(bid);
+    trackersToAdd.forEach(trackerToAdd => {
+      if (validVastTracker(trackers, trackerToAdd)) { trackers.push(trackerToAdd); }
+    });
+  });
+  const trackersMap = trackersToMap(trackers);
+  return (trackersMap.size ? trackersMap : null);
+};
+
+function validVastTracker(trackers, trackerToAdd) {
```

Comment:
```suggestion
function isValidVastTracker(trackers, trackerToAdd) {
```

the `is` prefix is a good indication that the function returns a bool

Reply:
done